tests/upgrades/test_errata.py

```diff
@@ -0,0 +1,260 @@
+            client_container_id,
+            'goferd -f',
+            **kwargs
+        )
+
+    def _check_package_installed(self, client_container_id, package):
+        kwargs = {'host': self.docker_vm}
+        docker_execute_command
+        installed_package = execute(
+            docker_execute_command,
+            client_container_id,
+            'rpm -q {}'.format(package),
+            **kwargs
+        )[self.docker_vm]
+        self.assertIn(package, installed_package)
+
+    def _install_package(self, client_container_id, package):
+        kwargs = {'host': self.docker_vm}
+        execute(
+            docker_execute_command,
+            client_container_id,
+            'yum install -y {}'.format(package),
+            **kwargs
+        )[self.docker_vm]
+        self._check_package_installed(client_container_id, package)
+
+    def _create_custom_rhel_tools_repos(self, client_os, product):
+        rhel_repo_url = os.environ.get('{}_CUSTOM_REPO'.format(client_os.upper()))
+
+        tools_repo_url = os.environ.get(
+            'TOOLS_{}'.format(client_os.upper()))
+        if None in [rhel_repo_url, tools_repo_url]:
+            raise ValueError('The Tools Repo URL/RHEL Repo url environment variable for '
+                             'OS {} is not provided!'.format(client_os))
+        tools_repo = entities.Repository(product=product,
+                                         content_type='yum',
+                                         url=tools_repo_url
+                                         ).create()
+        rhel_repo = entities.Repository(product=product,
+                                        content_type='yum',
+                                        url=rhel_repo_url,
+                                        ).create()
+        call_entity_method_with_timeout(rhel_repo.sync, timeout=1400)
+        return tools_repo, rhel_repo
+
+    @pre_upgrade
+    def test_pre_scenario_generate_errata_for_client(self):
+        """Create product and repo from which the errata will be generated for the
+        Satellite client or content host
+
+        :id: 88fd28e6-b4df-46c0-91d6-784859fd1c21
+
+        :steps:
+
+            1. Create  Life Cycle Environment, Product and Custom Yum Repo
+            2. Create custom tools, rhel repos and sync them
+            3. Create content view and publish it
+            4. Create activation key and add subscription.
+            5. Registering Docker Content Host RHEL7
+            6. Check katello agent and goferd service running on host
+            7. Generate Errata by Installing Outdated/Older Packages
+            8. Collect the Erratum list
+
+
+        :expectedresults:
+
+            1. The content host is created
+            2. errata count, erratum list will be generated to satellite client/content host
+
+        """
+        client_os = DISTRO_DEFAULT
+        # Step 1: Create  Life Cycle Environment, Product and Custom Yum Repo
```

Comment:
```suggestion
        # Step 1: Create  Life Cycle Environment, Product and Custom Yum Repo
```
please remove above docsting.